win32/devdoc/threadpool_win32.md

```diff
@@ -215,7 +215,7 @@ MOCKABLE_FUNCTION(, int, threadpool_timer_restart, TIMER_INSTANCE_HANDLE, timer,
 
 **SRS_THREADPOOL_WIN32_42_019: [** If `timer` is `NULL`, `threadpool_timer_restart` shall fail and return a non-zero value. **]**
 
-If timer state is `DESTROYING`, `threadpool_timer_restart` shall fail and return a non-zero value.
```

Comment:
>DESTROYING [](http://example.com/codeflow?start=19&length=10)

These states should be described at the top of the .md (Otherwise when reading this, I have no idea when we switch to the state and when we transition out of it) ...

Reply:
added